assets/queries/k8s/block_cluster_admin_rolebinding/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "block_cluster_admin_rolebinding",
+  "queryName": "Block Cluster Admin Rolebinding",
+  "severity": "LOW",
+  "category": null,
+  "descriptionText": "Ensure that the cluster-admin role is only used where required (RBAC)",
+  "descriptionUrl": "feature/block_cluster_admin_rolebinding"
```

Comment:
https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles
Still missing the URL :stuck_out_tongue_winking_eye:

Reply:
Sorry :P